docs/UserGuide.adoc

```diff
@@ -227,6 +227,35 @@ Format: `exit`
 Address book data are saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
+== Features(Coming in v2.0)
+
+=== Add relationship
+
+Enters the indices of the two people in between whom the user wants to add an relationship +
```

Comment:
If you look at the examples currently in the User Guide, you can see that the description should be more of a succinct explanation of what the feature achieves. For example, for the command `delete`, the description is "Deletes the specified person from the address book.", and the usage of the command is explained using the command `FORMAT` in the subsequent line. 

So for consistency's sake, I think it is better to follow this style as well when we update the User Guide
 
P/S It is "a relationship", not "an" :P